src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -1184,6 +1184,17 @@ public int hashCode() {
         return pattern.hashCode(); // enough for reasonable distribution
     }
 
+    /**
+     * {@return a string identifying this {@code MessageFormat}, for debugging}
+     */
+    @Override
+    public String toString() {
+        return
+            """
+            MessageFormat [locale: "%s", pattern: "%s"]
+            """.formatted(locale == null ? "null" : locale.getDisplayName(), toPattern());
```

Comment:
It would be more accurate if when locale ==null that null was not quoted in the string.
Seeing "null" would imply that the displayName of the locale was "null", when it was `null`.

Reply:
Hi Roger, addressed in https://github.com/openjdk/jdk/pull/17355/commits/70e0a175037ccd0215d76fb2fbfa8c91de291d41; would like to confirm the update is okay with you before integration.